pkg/errors/errors.go

```diff
@@ -124,3 +126,38 @@ func Cause(err error) error {
 
 	return err
 }
+
+// MultiError is an error that wraps multiple errors.
+type MultiError interface {
+	error
+	Unwrap() []error
+}
+
+// Join returns an error that wraps the given errors. Any nil error values are
+// discarded. Join returns nil if errs contains no non-nil values. The error
+// formats as the concatenation of the strings obtained by calling the Error
+// method of each element of errs and formatting like:
+//
+//	[first error, second error, third error]
+//
+// Note: aggregating errors should not be the default. Usually, return only the
+//
+//	first error, and only aggregate if there is clear value to the user.
```

Comment:
Looks like the formatting got a little messed up here?

Reply:
indeed

Reply:
fixed